.github/workflows/reusable-markdown-lint.yml

```diff
@@ -12,9 +12,6 @@ jobs:
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
-      - name: Install markdownlint
-        run: npm install -g markdownlint-cli
-
       - name: Run markdownlint
         run: |
-          markdownlint -c .github/config/markdown-lint-config.yml **/*.md
+          npx markdownlint-cli@0.43.0 -c .markdownlint.yml -p .gitignore **/*.md -i licenses/licenses.md
```

Comment:
intentionally one version behind to make sure the `customManager` added in renovate json works